### PR TITLE
replaced competition with contest, slight rephrases around it

### PR DESCRIPTION
**Pull request by mramendi**
These are all the substantial changes made in Chapter 3 for GSG, actually. (There will also be modularization and download link changes; we need to work out conditioning before link changes can be anywhere close to upstream repo)

### CODE REVIEW

File: optaplanner-docs/src/main/asciidoc/UseCasesAndExamples/ExamplesOverview/ExamplesOverview-section.adoc [hunk 1]
```diff
@@ -3,23 +3,23 @@
 
 Planner has several examples. In this manual we explain mainly using the _n_ queens example and cloud balancing example. So it is advisable to read at least those sections.
 
-The `Competition?` column in the following table identifies an example as being either realistic or unrealistic. A _realistic competition_ is _an official, independent competition_:
+Some of the examples solve problems that are presented in academic contests. The `Contest` column in the following table lists the contests. It also identifies an example as being either _realistic_ or _unrealistic_ for the putpose of a contest. A _realistic contest_ is _an official, independent contest_:
```
Comment (ge0ffrey) [inline optaplanner-docs/src/main/asciidoc/UseCasesAndExamples/ExamplesOverview/ExamplesOverview-section.adoc]: typo: putpose

Let's remove the entire sentence "It also identifies an example as being either _realistic_ or _unrealistic_ for the putpose of a contest." as the definition of "realistic" is immediately after.

Comment (mramendi) [reply optaplanner-docs/src/main/asciidoc/UseCasesAndExamples/ExamplesOverview/ExamplesOverview-section.adoc]: WIll fix typo but disagree about the sentence. Without this sentence, the definition of "realistic" simply comes out of nowhere. 

The sentence is a stylistic/logical connection between "table lists the contents" and "a realistic contest is...". We must first introduce the fact that a contest can be "realistic" or "unrealistic" and only then define the criteria for that.

Comment (ge0ffrey) [reply optaplanner-docs/src/main/asciidoc/UseCasesAndExamples/ExamplesOverview/ExamplesOverview-section.adoc]: Ok, although I think "for the purpose of the contest" is dead weight, I don't mind keeping it in. Will merge once typo is fixed.

**Comment by ge0ffrey** [comment]
@mramendi Can you add a commit to fix the typo? (to this branch on your fork, it will automatically appear in this PR) Then I 'll merge this PR.

**Comment by mramendi** [comment]
@ge0ffrey my apologies, I am up to my ears in a freeze prep so there was a delay, pushed a commit now.

**Comment by ge0ffrey** [comment]
Merged, thanks Misha :)

**Pull request merged**